tests/_tests_scripts/dl_z_mvp_mnist_metric_learning.py

```diff
@@ -0,0 +1,97 @@
+import os
+
+from torch.optim import Adam
+from torch.utils.data import DataLoader
+
+from catalyst import contrib as ctb, data, dl
+import catalyst.contrib.data.transforms as t
+from catalyst.contrib.models.simple_conv import SimpleConv
+from catalyst.core.callbacks import ControlFlowCallback
+import catalyst.data.sampler_inbatch as si
+from catalyst.dl.callbacks.metrics.cmc import CMCScoreCallback
+
+
+def run_ml_pipeline(sampler_inbatch: si.InBatchTripletsSampler) -> float:
```

Comment:
btw, could we just use it as `data. InBatchTripletsSampler `? as far as you have imported `from catalyst import contrib as ctb, data, dl`

Reply:
done

Reply:
done